src/oas.md

```diff
@@ -44,6 +44,31 @@ Each template expression in the path MUST correspond to a path parameter that is
 
 The value for these path parameters MUST NOT contain any unescaped "generic syntax" characters described by [RFC3986](https://tools.ietf.org/html/rfc3986#section-3): forward slashes (`/`), question marks (`?`), or hashes (`#`).
 
+The path templating expression is defined by the following [ABNF](https://tools.ietf.org/html/rfc5234) syntax
+
+```abnf
+; OpenAPI Path Templating ABNF syntax
+path-template                  = path [ query-marker query ]
```

Comment:
I have removed the fragment part from the original work but left the query parameter part:
- fragments: never go to the service, no point in adding them to the spec anyway
- query parameters: probably shouldn't be here either, but wanted to triple confirm we want to remove it since some descriptions use it to describe "constant query parameters", and removing those would technically break them.

Reply:
I'd remove the query part because the specification text clearly talks about paths, and section [4.8.12.1 Parameter Locations](https://spec.openapis.org/oas/v3.1.1.html#parameter-locations) clearly distinguishes path parameters and query parameters.

Making this explicit in the ABNF may prevent future loose interpretation of the spec.

Reply:
Thanks for confirming, removed